test/systemtests/docker_test.go

```diff
@@ -451,22 +451,25 @@ func (d *docker) startNetmaster() error {
 func (d *docker) cleanupMaster() {
 	logrus.Infof("Cleaning up master on %s", d.node.Name())
 	vNode := d.node.tbnode
-	vNode.RunCommand("etcdctl rm --recursive /contiv")
-	vNode.RunCommand("etcdctl rm --recursive /contiv.io")
-	vNode.RunCommand("etcdctl rm --recursive /docker")
-	vNode.RunCommand("curl -X DELETE localhost:8500/v1/kv/contiv.io?recurse=true")
-	vNode.RunCommand("curl -X DELETE localhost:8500/v1/kv/docker?recurse=true")
+	vNode.RunCommand(`etcdctl rm --recursive /contiv || true &
```

Comment:
I realize this is not new code you're adding, but what is the purpose of recursively deleting a key from the command-line and also recursively deleting it using the HTTP API?